server.js

```diff
@@ -1,7 +1,8 @@
 var Hapi = require('hapi');
 var Hoek = require('hoek');
 
-var log = require('./logger.js');
+// Initialize the environment before anything else happens
+var env = require('./lib/environment');
```

Comment:
This is an important fix that isn't detailed in the commit log. It closes https://github.com/mozilla/publish.webmaker.org/issues/63